content-src/components/ASRouterAdmin/ASRouterAdmin.jsx

```diff
@@ -239,13 +239,14 @@ export class ASRouterAdmin extends React.PureComponent {
   renderTargetingParameters() {
     // There was no error and the result is truthy
     const success = this.state.evaluationStatus.success && !!this.state.evaluationStatus.result;
+    const result = JSON.stringify(this.state.evaluationStatus.result, null, 2) || "undefined";
```

Comment:
maybe `"(Empty result)"` or something? `undefined` kinda looks like something is broken